android/app/src/main/java/com/reactnativeapp/MainActivity.java

```diff
@@ -12,4 +25,47 @@
     protected String getMainComponentName() {
         return "reactNativeApp";
     }
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+
+        setupEventSender();
+    }
+
+    private void setupEventSender() {
```

Comment:
Isn't this quite a lot of logic for one logging library initialization? This is not very reusable and I would expect that these details are going to be hidden in the logging library itself. Go for it, please...

Reply:
This is how the logging library is initialized in the core kiwi.com Android app. It should induce as similar behavior as in production as possible.

Reply:
Do you realize that this is just a copy-paste from `logging/EventHelper.java` and therefore it's going to be fragile?

Reply:
So, I think what @mrtnzlml wants is to be able to call LoggingLibrary.setupEventSender() from onCreate instead of defining the setupEventSender in MainApplication.java. 

Changes in this code would in the future have to be added both in Kiwi native app and react-native-app. 

Is there some way to do this?

Reply:
OK, I'll move that method to logging library but it will still has to have this signature:

```
EventSender.initialize(Application application, AppConfig appConfig, String visitorId, boolean isDebug);
```

Reply:
Done.